### DISCUSSION TITLE
I need help for loading background image

### DISCUSSION THREAD

**Discussion by dodero4** [ipxe/ipxe#1124] [General]
Hi, I haven't used my ipxe server for 2 years, I remember being able to make the background image work without problems but now I can't understand why it doesn't work and I receive a connection timeout in the tftp phase, without the image background my menu loads without problems I'll go into detail:

In my menu I wrote like this:

`console --picture ipxe.png`

as well as having enabled

#define IMAGE_PNG /* PNG images */
#define CONSOLE_CMD /* Console commands */

inside general.h

And

#define CONSOLE_FRAMEBUFFER /* Graphical framebuffer console */

inside console.h

as also described in the article

https://ipxe.org/buildcfg/console_framebuffer

If I can be of help in any way by adding further information, I remain available and thanks in advance

**Comment by NiKiZe** [comment]
Post actual error message (including url to ipxe.org), follow and read that url, verify any suggestions it gives.

**Reply by dodero4** [reply]
the error I get is this below

![IMG_1283](https://github.com/ipxe/ipxe/assets/156615322/e2bf7c4c-eed2-4adb-8cc0-6bd11e890620)

and inside my boot menu the code is this:

console --picture http://boot.ipxe.org/texture.png

Going back to 2 years ago I had only entered:

console --picture ipxe.png

and it worked without any problems as per the image attached below

![camphoto_824023566](https://github.com/ipxe/ipxe/assets/156615322/1dd4127d-7bdf-4ceb-a63e-0ddc27459893)

**Reply by NiKiZe** [reply]
Did you follow the URL that iPXE tells you to go look at?

**Reply by dodero4** [reply]
I think so, I'm not very expert but completely self-taught, I do my best, if you can tell me something precise I'll check and I can tell you more precisely, the strange thing in the code as I've always used that is

console --image ipxe.png worked before, and now on this new openmediavault machine it doesn't

**Comment by dodero4** [comment]
@NiKiZe 
Can you help me?

**Reply by Supermathie** [reply]
"connection timed out" → the problem is ipxe can't retrieve the image

**Comment by ZeroQI** [comment]
https://ipxe.org/err/4c1160 so message indicate a DNS issues
iPXE 1.0.0+ so not utilising latest which is 1.2.1+, so did not follow the error message...
